TS_lobocTests/is1_22.loboc

```diff
@@ -0,0 +1,14 @@
+// ISEQ1: Test addition
```

Comment:
@tylersheltonnn, could you update the comments to reflect the contents of the file?

Reply:
Did it!